src/route/builder.rs

```diff
@@ -403,25 +411,15 @@ impl RouteMessageBuilder<IpAddr> {
         mut self,
         addr: IpAddr,
     ) -> Result<Self, InvalidRouteMessage> {
-        self.set_address_family_from_ip_addr(addr);
-        match self.message.header.address_family {
-            AddressFamily::Inet => {
-                if addr.is_ipv6() {
-                    return Err(InvalidRouteMessage::Gateway(addr));
-                };
-            }
-            AddressFamily::Inet6 => {
-                if addr.is_ipv4() {
-                    return Err(InvalidRouteMessage::Gateway(addr));
-                };
-            }
-            af => {
-                return Err(InvalidRouteMessage::AddressFamily(af));
+        use AddressFamily::*;
```

Comment:
Please do not use ` use XXX::*`, it will cause problem.
For example, the `android` does not have `AddressFamily::Mpls`. If I have a match arm like:

```
Mpls => {do something}
```

This Mpls will be variable instead of enum variant.

Reply:
Fixed.